gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_0_0/schema.yml

```diff
@@ -118,4 +118,14 @@ databaseChangeLog:
                   name: type
```

Comment:
addPrimaryKey detected in liquibase changelog 💥. Please create the primary key directly with the create table statement.